src/views/Privacy.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <right-section></right-section>
     <left-section></left-section>
-    <div class="container wrapper">
+    <div class="container wrapper mt-2">
```

Comment:
IMHO we should later remove that container.wrapper stuff and replace it with something proper also remove boostrap at all

Reply:
I agree we can do it at a later stage